apps/wasm-graphcompiler-tvm/README.md

```diff
@@ -0,0 +1,191 @@
+# WebAssembly GraphCompiler for Deep Learning Framework with TVM Runtime
+
```

Comment:
Shall we rename it to wasm-standalone? since what we are doing is compiling to a standalone wasm application

Reply:
Sure